spec/lib/upload_creator_spec.rb

```diff
@@ -441,4 +441,22 @@ def expect_no_public_context_uploads_to_be_secure
       end
     end
   end
+
+  describe '#should_downsize?' do
+    context "GIF images" do
+      let(:gif_file) { file_from_fixtures("huge.jpg") }
```

Comment:
Is it a GIF or a JPG? 🤔

Reply:
I did not want to add a GIF image to fixtures, but eventually did to get rid of the other stubs too.